dm/loader/checkpoint.go

```diff
@@ -470,19 +466,44 @@ func (cp *RemoteCheckPoint) String() string {
 	return string(bytes)
 }
 
+type lightingLoadStatus int
+
+const (
+	lightningStatusInit lightingLoadStatus = iota
+	lightningStatusRunning
+	lightningStatusFinished
+)
+
+func (s lightingLoadStatus) String() string {
+	switch s {
+	case lightningStatusInit:
+		return "init"
+	case lightningStatusRunning:
```

Comment:
i can't find when we update tasksatus to `lightningStatusRunning`😅

Reply:
Add in the beginning of runLightning.